zombies.js

```diff
@@ -79,6 +95,119 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+class Player {
+  constructor(name, health, strength, speed) {
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
+    this.speed = speed;
+    this.isAlive = true;
+    this.equipped = false;
+    this._pack = [];
+    this._maxHealth = 100;
+  }
+  getPack() {
+    return this._pack;
+  }
+
+  getMaxHealth() {
+    return this._maxHealth;
+  }
+
+  checkPack() {
+    let pack = this.getPack();
+    console.log(pack);
+    // does it need return before console.log?
+    // what's the diff between this.getPack() vs Player.getPack? Should we use one over the other?
+  }
+
+  takeItem(item) {
+    if (this._pack.length < 3) {
+      this._pack.push(item);
```

Comment:
This is missing both a console.log message with the player's and item's name, and it should return true.